static/app/views/issueList/searchBar.tsx

```diff
@@ -145,23 +141,8 @@ function IssueListSearchBar({organization, tags, ...props}: Props) {
     ]
   );
 
-  const getTagValues = useCallback(
```

Comment:
Moved this function into the util file below so it could be reused in the all events component.